README.md

```diff
@@ -1,14 +1,67 @@
-# Cosmos
+<p align="center">
 
-[![Build status](https://ci.appveyor.com/api/projects/status/kust7g5dlnykhkaf/branch/master?svg=true)](https://ci.appveyor.com/project/CosmosOS/cosmos/branch/master)
-[![Join the chat at (https://discord.com/invite/kwtBwv6jhD](https://img.shields.io/discord/833970409337913344)](https://discord.com/invite/kwtBwv6jhD)
+  <img src="https://cosmosos.github.io/images/logo.png" width="25%" />
 
-Cosmos is an operating system "construction kit" made for Visual Studio 2022. Build your own OS using managed languages such as C#, VB.NET, and more!
+</p>
 
-For instructions on how to install and use Cosmos, please visit the [Cosmos website](http://www.gocosmos.org).
+<h1 align="center">Cosmos</h1>
+
+<p align="center">
+
+  <a href="https://ci.appveyor.com/api/projects/status/kust7g5dlnykhkaf/branch/master">
+    <img src="https://ci.appveyor.com/api/projects/status/kust7g5dlnykhkaf/branch/master?svg=true" />
+  </a>
+
+  <img src="https://img.shields.io/github/languages/code-size/CosmosOS/Cosmos" />
+  <img src="https://img.shields.io/github/downloads/CosmosOS/Cosmos/total" />
+
+  <a href="https://github.com/CosmosOS/Cosmos/releases/latest">
+    <img src="https://img.shields.io/github/v/release/CosmosOS/Cosmos" />
+  </a>
+
+  <a href="https://github.com/CosmosOS/Cosmos/blob/master/LICENSE.txt">
+    <img src="https://img.shields.io/github/license/CosmosOS/Cosmos" />
+  </a>
+
+  <a href="https://github.com/CosmosOS/Cosmos/issues">
+    <img src="https://img.shields.io/github/issues/CosmosOS/Cosmos" />
+  </a>
+
+  <a href="https://discord.com/invite/kwtBwv6jhD">
+    <img src="https://img.shields.io/discord/833970409337913344" />
+  </a>
+
+</p>
+
+<hr/>
 
+Cosmos (C# Open Source Managed Operating System) is an operating system development kit which uses dotnet, along with the custom IL2CPU compiler to convert (most) C# code into a working bare-metal OS.
+Despite C# in the name, any .NET based language can be used, which includes: VB.NET, IronPython, F# and more. Cosmos itself and the kernel routines are primarily written in C#, and thus the Cosmos name.
+
+In a project, cosmos can be thought of as a compiler and a sort-of standard library for a project. It gives the user access to often hard to find or otherwise difficult to understand tools.
+
+<hr/>
+
+## Some of the included features
+
+These are some of the (optional) features that can be used in cosmos:
+
+- Low level assembly access and pointer memory control
+- A basic (and unstable at the moment) filesystem
+- Most fretures found in the .NET corlib
+- A CPU/FPU accelerated math library
+- A basic graphics interface
+- A basic network interface
+- A basic audio interface
+> Note: Use [embeded resources](https://cosmosos.github.io/articles/Kernel/ManifestResouceStream.html) instead of the FS for now.
```

Comment:
GitHub Flavored Markdown has a fancy way of creating these:
```md
> **Note**
> Use [embeded resources](https://cosmosos.github.io/articles/Kernel/ManifestResouceStream.html) instead of the FS for now.
```
turns into:
> **Note**
> Use [embeded resources](https://cosmosos.github.io/articles/Kernel/ManifestResouceStream.html) instead of the FS for now.

Also, I suggest adding a heads up that these resources are baked in into the kernel image, and remain in memory for the lifespan of the kernel, meaning they cannot be unloaded.